pkg/boot/bzimage/bzimage_test.go

```diff
@@ -205,18 +205,20 @@ func TestInitRAMFS(t *testing.T) {
 	Debug = t.Logf
 	cpio.Debug = t.Logf
 	for _, bz := range []string{"testdata/bzImage", "testdata/bzimage-64kurandominitramfs"} {
-		initramfsimage, err := os.ReadFile(bz)
-		if err != nil {
-			t.Fatal(err)
-		}
-		var b BzImage
-		if err := b.UnmarshalBinary(initramfsimage); err != nil {
-			t.Fatal(err)
-		}
-		s, e, err := b.InitRAMFS()
-		if err != nil {
-			t.Fatal(err)
-		}
-		t.Logf("Found %d byte initramfs@%d:%d", e-s, s, e)
+		t.Run(bz, func(t *testing.T) {
```

Comment:
usually, people put a readable description to this field,  but I guess the file path itself it kinda self-explanatory in some way.. [abrender](https://github.com/abrender)

If you feel like writing a bit more code and have time, sth like this is worth considering, 

```
for _, tt := range []struct {
    name string
    path string
}{
    {
        name: "basic bzImage",
        path: "testdata/bzImage",
    },
    { 
       name: "a little larger bzImage, 64k random generated image",
       path: "testdata/bzimage-64kurandominitramfs",
     },
}{
      {copy over existing block}
}
```

Reply:
Thanks for the suggestion! I was thinking of doing that but I thought that file names were enough for the user to distinguish between which file caused the error.

I've updated the pull request to add subtests to all of the tests (except `TestAddInitRAMFS`) and I've incorporated your suggestion, but moved the slice of name/path data to be a package level variable so that it can be reused in each test. Thoughts?